src/interactions/find-all.js

```diff
@@ -6,7 +6,7 @@ import { computed } from './helpers';
  * matching the provided selector.
  *
  * ``` javascript
- * let $listItems = await new Interactor('ul').findAll('li');
+ * let $listItems = new Interactor('ul').findAll('li');
```

Comment:
The methods `find` and `findAll` still needs to `await`. The property creators do not. You can see in the two function bodies for each one returns a convergence and the other returns a computed element.

The methods are used to wait for elements to exist, and the created properties can be used inside convergent assertions.